main.py

```diff
@@ -16,6 +16,5 @@
 # show_version: bool = __argv.version
 
 hello: str = f"hi{f', {username}' if username else ''}"
-# version: str = f" - python version : {sys.version_info.major}.{sys.version_info.minor}" if show_version else ""
```

Comment:
why are you printing the python version?

Reply:
It was just for testing. I'll remove for sure

Reply:
is your code complete yet?